aries/aries_vcx_core/src/wallet/entry_tag.rs

```diff
@@ -77,3 +110,42 @@ impl From<EntryTags> for Vec<EntryTag> {
         value.inner
     }
 }
+
+pub fn pair_into_entry_tag(pair: (String, String)) -> EntryTag {
```

Comment:
In these cases, how do you decide whether this should be standalone function, a static method or a `From` impl? I am not sure I see the pattern.

Reply:
Changed into a static method for consistency.